### ISSUE TITLE
[Feature Request] Option to Hide Side Bar in Windowed Mode

### ISSUE THREAD

**Issue by tacosuzy** [linuxmint/hypnotix#44]
This isn't a major necessity but just a casual request. I do things in windowed mode a lot and would find it convenient to be able to hide the left content selection sidebar and maybe even getting the video to go to the resolution with resize options if needed.

**Comment by clefebvre** [comment]
You want it to use the entire window but without the window going fullscreen? Some kind of a theater mode, is that it?

**Comment by tacosuzy** [comment]
Yes, you worded it perfectly. Just an idea. I've recently switched back to windows for a short time so there is no rush just wanted to pitch an idea.
